py/selenium/webdriver/common/virtual_authenticator.py

```diff
@@ -153,7 +153,7 @@ def create_resident_credential(
 
         :Args:
           - id (bytes): Unique base64 encoded string.
-          - rp_id (str): Relying party identifier.
+          - rp_id (str): Relying on party identifier.
```

Comment:
This is a proper name https://www.w3.org/TR/webauthn-2/#relying-party-identifier
update the others as well.
```suggestion
          - rp_id (str): Relying Party Identifier.
```